### PR TITLE
addeded deletensgflowlog separately for customer use

### PR DESCRIPTION
**Pull request by krishmi**

### CODE REVIEW

File: DeleteNSGFlowLogs.ps1 [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        }
+    }
+}
+
+
+#endregion
+
+try
+{
+	$configPath = Read-ValuesIgnoringPreviousEntries("Please enter the path to scope selecting config file:")
```
Comment (nitigupta16) [inline DeleteNSGFlowLogs.ps1]: Input text is confusing - Suggestion " Please enter the path to select config file"

File: DeleteNSGFlowLogs.ps1 [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	$subIdRegion = Get-Content -Path $configPath | ConvertFrom-Json  -AsHashtable -ErrorAction SilentlyContinue
+}
+catch
+{
+	Write-Host "Config file in incorrect json format please format it correctly" -ForegroundColor Red
```
Comment (nitigupta16) [inline DeleteNSGFlowLogs.ps1]: NIt: "Config file is in incorrect json format, please format it correctly"

File: DeleteNSGFlowLogs.ps1 [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+	return
+}
+
+if ($null -eq $subIdRegion)
+{
+	Write-Host "Config file in incorrect json format please format it correctly" -ForegroundColor Red
```
Comment (nitigupta16) [inline DeleteNSGFlowLogs.ps1]: same as above

File: DeleteNSGFlowLogs.ps1 [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+function Delete-NSGFlowLogs()
+{
+    Write-Host "Getting all disabled NSG flowlogs in region:" $region "and subscription:" $subscriptionId -ForegroundColor Blue
+    $flList =  Get-AzNetworkWatcherFlowLog -Location $region
+    $disabledNSGFlList =  Filter-DisabledNSGFlowLogs $flList
+    $proceed = Read-ValuesIgnoringPreviousEntries("Proceed with deletion of flowlogs?(y/n)")
+
+    if ($proceed -eq 'y')
```
Comment (nitigupta16) [inline DeleteNSGFlowLogs.ps1:96]: can we make it case isensitive?

File: DeleteNSGFlowLogs.ps1 (hunk 1 above)
Comment (krishmi) [reply DeleteNSGFlowLogs.ps1]: done

File: DeleteNSGFlowLogs.ps1 (hunk 3 above)
Comment (krishmi) [reply DeleteNSGFlowLogs.ps1]: done

**Review by nitigupta16 — APPROVED** [review]

**Pull request merged**